lang/eo/strings.po

```diff
@@ -1340,75 +1350,75 @@ msgstr "Malnova&nbsp;Domo"
 
 #: script/world.js:124
 msgid "A&nbsp;Damp&nbsp;Cave"
-msgstr ""
+msgstr "Malseka Kaverno"
```

Comment:
Sorry about that, I'm using PoEdit which doesn't display non-breaking spaces any differently to normal spaces so I didn't notice them. I'll make sure to watch out for them.